r/R/dataset-format.R

```diff
@@ -133,10 +133,36 @@ CsvFileFormat$create <- function(...,
   schema_names <- names(schema)
 
   if (!is.null(schema) & !identical(schema_names, column_names)) {
+    missing_from_schema <- setdiff(column_names, schema_names)
+    missing_from_colnames <- setdiff(schema_names, column_names)
+    message_colnames <- NULL
+    message_schema <- NULL
+    message_order <- NULL
+
+    if (length(missing_from_colnames) > 0) {
+      message_colnames <- paste(
+        oxford_paste(missing_from_colnames, quote_symbol = "`"),
+        "not present in `column_names`"
+      )
+    }
+
+    if (length(missing_from_schema) > 0) {
+      message_schema <- paste(
+        oxford_paste(missing_from_schema, quote_symbol = "`"),
+        "not present in schema"
```

Comment:
Minor: should `schema` here also be in backticks? `column_names` is up above, though on the other hand, I think it's a little bit weird we treat column names and arguments the same(?) Do we already (or does the tidyverse style guide) have a clear decision on how to quote column names, arguments, etc.?

Reply:
Yeah, I was thinking of schema as in the type of object here instead of the parameter name, but will update as backticks remove ambiguity.